spec/models/system_announcement_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe SystemAnnouncement, type: :model do
+  context 'when title is not present' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.